src/cunumeric/unary/unary_op_util.h

```diff
@@ -57,6 +58,7 @@ enum class UnaryOpCode : int {
   LOG1P       = CUNUMERIC_UOP_LOG1P,
   LOG2        = CUNUMERIC_UOP_LOG2,
   LOGICAL_NOT = CUNUMERIC_UOP_LOGICAL_NOT,
+  MODF        = CUNUMERIC_UOP_MODF,
   NEGATIVE    = CUNUMERIC_UOP_NEGATIVE,
   POSITIVE    = CUNUMERIC_UOP_POSITIVE,
```

Comment:
FREXP and MODF are not handled in the `op_dispatch` switch below.

Reply:
I guess it doesn't matter, because these are handled specially. However, clang still complains:

```
./cunumeric/unary/unary_op_util.h:82:11: warning: enumeration values 'FREXP' and 'MODF' not handled in switch [-Wswitch]
  switch (op_code) {
          ^
./cunumeric/unary/unary_op_util.h:82:11: note: add missing switch cases
  switch (op_code) {
          ^
```

Reply:
I added the missing cases with an assertion, so the compiler would no longer complain about them.